app/index.js

```diff
@@ -112,6 +113,23 @@ export default class Root extends React.Component {
 		}
 	}
 
+	// It should run only once
+	encryptMigratedData = async() => {
+		try {
+			const encryptMigration = await UserPreferences.getBoolAsync('encryptMigration');
+
+			if (!encryptMigration) {
+				// Encrypt the migrated data
+				await UserPreferences.encryption.encrypt();
```

Comment:
Can you make this call inside `UserPreferences` and export a new function called `encrypt`?